shells/firefox/webpack.config.js

```diff
@@ -9,8 +9,10 @@
  */
 'use strict';
 
+var __DEV__ = process.env.NODE_ENV !== 'production';
+
 module.exports = {
-  // devtool: 'cheap-module-eval-source-map',
+  devtool: __DEV__ ? 'cheap-module-eval-source-map' : false,
```

Comment:
Firefox actually has rather poor sourcemap support generally, which is why I disabled source maps even in dev.